mix.exs

```diff
@@ -47,20 +47,19 @@ defmodule Membrane.RTSP.MixProject do
 
   def application do
     [
-      extra_applications: [:logger],
-      mod: {Membrane.RTSP.Application, []}
+      extra_applications: [:logger]
     ]
   end
 
   defp deps do
     [
-      {:bunch, "~> 1.0"},
+      {:bunch, "~> 1.3"},
       {:membrane_protocol_sdp, "~> 0.1.0"},
```

Comment:
There is no longer `membrane_protocol_sdp`. Not sure if we want to update to `ex_sdp` at this moment

Reply:
As discussed, I am leaving an issue about it